app/controllers/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UsersController < ApplicationController
     def show
-      @user = current_user
+      @user  = current_user
+      @items = Item.all
```

Comment:
This should be @items = @user.items so we can narrow the Item scope to the current user.